### PR TITLE
[CI:DOCS] minor man page cleanup

### PR DESCRIPTION
**Pull request by edsantiago**
Followup to #21285. I hope this is easier to review
than a re-push of that one.

Signed-off-by: Ed Santiago <santiago@redhat.com>
```release-note
ongoing man page cleanup, maybe worth a broad release note but certainly not for just this PR
```

### CODE REVIEW

File: docs/source/markdown/podman-wait.1.md.in [hunk 1]
```diff
@@ -72,7 +72,7 @@ $ podman wait mywebserver myftpserver
 125
 ```
 
-Wait for does-not-exist container to exit, but do not fail if container does not exist.
+Wait for named container to exit, but do not fail if container does not exist.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-wait.1.md.in:75]:
```suggestion
Wait for the named container to exit, but do not fail if the container does not exist.
```

File: docs/source/markdown/podman-wait.1.md.in [hunk 2]
```diff
@@ -53,13 +53,13 @@ $ podman wait --latest
...
 
-Wait up to 2 seconds the specified container to exit.
+Wait for container to exit, checking every two seconds.
 ```
 $ podman wait --interval 2s mywebserver
 0
 ```
 
-Wait for the specified container to exit with a failure.
+Wait for container by ID. This container exits with error status 1:
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-wait.1.md.in:62]:
```suggestion
Wait for the container by ID. This container exits with error status 1:
```

File: docs/source/markdown/podman-wait.1.md.in [hunk 3]
```diff
@@ -53,13 +53,13 @@ $ podman wait --latest
 0
 ```
 
-Wait up to 2 seconds the specified container to exit.
+Wait for container to exit, checking every two seconds.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-wait.1.md.in:56]:
```suggestion
Wait for the container to exit, checking every two seconds.
```

**Review by giuseppe — APPROVED** [review]
LGTM

**Review by vrothberg — APPROVED** [review]
/lgtm

**Pull request merged**